hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeAdminDefaultMonitor.java

```diff
@@ -265,6 +266,7 @@ private void check() {
               // to track maintenance expiration.
               dnAdmin.setInMaintenance(dn);
             } else {
+              inValidState  = true;
               Preconditions.checkState(false,
```

Comment:
indentation

Reply:
What's the indentation issue here?

Reply:
`"Node %s is in an invalid state! "
                      + "Invalid state: %s %s blocks are on this dn.",`

Reply:
>"Node %s is in an invalid state! " + "Invalid state: %s %s blocks are on this dn."

I am not sure if there is really an indentation issue here. I have not added this change here. To correct existing indentations, a separate JIRA can be created to track and correct this on module level if required.